core/common/src/main/java/alluxio/underfs/UnderFileSystem.java

```diff
@@ -68,7 +68,7 @@ private Cache() {}
        * @param ufsConf the ufs configuration
        * @return the UFS instance
        */
-      UnderFileSystem get(String path, Object ufsConf) {
+      UnderFileSystem get(String path, Map<String, String> ufsConf) {
```

Comment:
as I commented in below, I agree with the change and suggest to have a separate PR on this.